scripts/build/toolchain/termux_setup_toolchain_28c.sh

```diff
@@ -226,6 +226,9 @@ termux_setup_toolchain_28c() {
 		# This seem to be needed when building rust
 		# packages
 		echo 'INPUT(-lunwind)' > $dir/libgcc.a
+		# Warning: PHP Startup: Unable to load dynamic library... libdl.a is a stub --- use libdl.so instead
```

Comment:
Please also bump the revision of the toolchain in `termux_step_setup_toolchain.sh`.